pkg/rpc/context.go

```diff
@@ -82,7 +82,7 @@ var SourceAddr = func() net.Addr {
 	return nil
 }()
 
-var enableRPCCompression = envutil.EnvOrDefaultBool("COCKROACH_ENABLE_RPC_COMPRESSION", false)
+var enableRPCCompression = envutil.EnvOrDefaultBool("COCKROACH_ENABLE_RPC_COMPRESSION", true)
```

Comment:
Perhaps. I'm a bit on the fence. It currently only takes effect at startup, so we'd have to rework things a bit, and even we'd only be able to make it take effect on new outgoing connections. Modifying the created servers seems like too much change.